app/javascript/components/legacy/nonprofits/donate/amount-step.tsx

```diff
@@ -1,52 +1,67 @@
-import React, { useContext } from 'react';
+import React, { Dispatch, SetStateAction, useContext, useState } from 'react';
 import { useId } from "@reach/auto-id";
 import { Money } from '../../../../common/money';
 import { Formik, useFormikContext } from 'formik';
 import { ActionType, DonationWizardContext } from './wizard';
-declare const I18n: any;
+import { useIntl } from "../../../intl";
+import { format } from 'sinon';
+import { WizardContext } from '../../_dependencies/ff-core/wizard';
+import BigNumber from 'bignumber.js';
+
 interface AmountStepProps {
-	amount: Money|null;
+	amount: Money | null;
 	amountOptions: Money[];
+	stateDispatch: (action: ActionType) => void;
+	currencySymbol: string;
+	singleAmount: string | null;
```

Comment:
This should be `Money | null`. Everything else here uses Money so let's be consistent.